src/components/Layout/AuthModal/components/Register/Register.jsx

```diff
@@ -225,10 +233,15 @@ export function Register({ switcher }) {
     }
 
     function validateForm() {
+        let validateSp = " ";
+        if (!isSponsor) {
+            validateSp = validateSpecialization();
+        }
+
         return (
             validateFirstName() &&
             validateLastName() &&
-            validateSpecialization() &&
+            validateSp &&
```

Comment:
You can delete rows 236-239
```suggestion
            (isSponsor || validateSpecialization()) && 
```